Program.cs

```diff
@@ -120,6 +154,20 @@ private static void RunUGL()
     private static string GetClientProcessPath(string directory, string file)
         => $"{Resources}\\{Binaries}\\{directory}\\{file}";
 
+    private static int? ShowIncompatibleGPUMessage(string[] selections)
+    {
+        return AdvancedMessageBoxHelper.ShowMessageBoxWithSelection(
+            string.Format(
+                "The client has detected an incompatibility between your graphics card and the DirectX11 and OpenGL versions of the CnCNet client.\n\n" +
```

Comment:
This line is much longer than the lines below, making the message excessively wide and its layout weird. Split it from the middle, like the line below.